packages/launch/src/utils.ts

```diff
@@ -22,7 +22,7 @@ export const isDefaultSiteTitle = ( {
 		: new RegExp( DEFAULT_SITE_NAME, 'i' ).test( currentSiteTitle );
 
 export const isValidSiteTitle = ( title?: string ): boolean =>
-	title !== '' && ! isDefaultSiteTitle( { currentSiteTitle: title } );
+	title !== '' && ! isDefaultSiteTitle( { currentSiteTitle: title, exact: true } );
```

Comment:
This change looks right. Since now we're calling everywhere `isDefaultSiteTitle` with the `exact: true` option, maybe that's not needed anymore 🤷

Reply:
Probably right but it could be nice to have that optional flexibility on this function? I can remove it if we feel it's completely redundant now. What do you think?

Reply:
I think this has been added when we didn't have the site title step persistent to prevent the layout from jumping.

There is one very specific case I have in mind: if a user adds some character to the default title, then closes the flow and opens it on a new device/browser (or clears local storage data). Only in this case when we might want to do the non-exact matching to show them the title step. But the scenario is very unlikely so we could remove this without any concern.